src/client/components/QuizAnswers/QuizAnswers.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './QuizAnswers.styles.css';
+
+function QuizAnswers({ isAgreementQuestion, firstAnswer, secondAnswer }) {
+  const [selected, setSelected] = useState('');
+  const handleSelectChange = (event) => {
+    const {
+      target: { value },
+    } = event;
+    setSelected(value);
+  };
+
+  return (
+    <form>
+      {isAgreementQuestion ? (
+        <div className="inline">
+          <label className="radio radio_before">
+            <span className="radio_label">Disagree</span>
+            <span className="radio_input">
+              <input
+                type="radio"
+                value="disagree"
+                checked={selected === 'disagree'}
+                onChange={(event) => handleSelectChange(event)}
```

Comment:
It could also be written like this as it will be invoked with the event :) 
```suggestion
                onChange={handleSelectChange}
```